.github/workflows/publish.yml

```diff
@@ -0,0 +1,119 @@
+name: Publish
+
+# on:
+#   # This allows us to run this workflow manually from the Actions tab
+#   workflow_dispatch:
+#     inputs:
+#       version-strategy:
+#         type: choice
+#         description: 'Version strategy'
+#         required: true
+#         options:
+#             - 'patch'
+#             - 'minor'
+#             - 'major'
+#         default: patch
+#       title:
+#         type: string
+#         description: 'Release title'
+#         required: true
```

Comment:
**Suggestion:** we can leave this uncommented, this specific trigger won't take effect until we merge this to the main branch.

Reply:
Yessir 9c21b5743b637673bada7d3286551880b9e9d7f7